src/vue/backend/src/views/guided-setup/Agents.vue

```diff
@@ -64,12 +70,14 @@ export default {
             progressStepperUpdate: 'progressStepper/progressStepperUpdate'
         }),
         async saveHandler () {
+            this.formDisabled = true
             if (this.formChanges) {
                 const { status } = await this.agentSettingsRepository.post({ enabled: this.localStateValues.enabled }, 'enable')
                 if (status === 204) {
                     this.saveAction()
                     this.$router.push({ path: this.continuePath })
                 } else {
+                    this.formDisabled = false
```

Comment:
This should be before the status checks, as we want the forms to be enabled on both success and failure
*Repeat for all instances